docs/concepts/codesharing.md

```diff
@@ -11,24 +11,24 @@ ms.technology: .net-core-technologies
 ms.devlang: dotnet
 ms.assetid: d4c31c29-e977-48f2-b4c7-0335ef50d440
 ---
+# Code Sharing with .NET Core and the .NET Standard Library #
```

Comment:
this one doesn't seem to be causing issues but I'd add the line back just in case between metadata and title